docs/source/internals.rst

```diff
@@ -169,9 +168,9 @@ Pilot (in session profile, all optional)
 
 ::
 
-   staging_in_start    : pilot level staging request starts         (uid: pilot, msg: did, [PILOT-DS])
-   staging_in_fail     : pilot level staging request failed         (uid: pilot, msg: did, [PILOT-DS])
-   staging_in_stop     : pilot level staging request stops          (uid: pilot, msg: did, [PILOT-DS])
+   staging_in_start    : pilot level staging request starts         (uid: pilot, msg: did, [PILOT-DS]) -- agent; client (rp.session.<SID>.prof)
+   staging_in_fail     : pilot level staging request failed         (uid: pilot, msg: did, [PILOT-DS]) -- agent; client (rp.session.<SID>.prof)
+   staging_in_stop     : pilot level staging request stops          (uid: pilot, msg: did, [PILOT-DS]) -- agent; client (rp.session.<SID>.prof)
```

Comment:
`<SID>` would be a full name including `rp.session`

Reply:
Right. Went with `*` so not to have to explain what a SID is.